README.md

```diff
@@ -256,9 +256,49 @@ This will match the export names `"foo"` and `"bar"` and will be treated as the
 
 > Note: Wildcard Exports currently only supports the exports key `"./*"`, which will match all the available entries.
 
+### `bin` Field Convention
```

Comment:
Can we move this section up a bit, under `Special Exports Conventions`?

Reply:
Moved it!